### DISCUSSION TITLE
kubevpn-bin package added to Arch AUR

### DISCUSSION THREAD

**Discussion by jonstelly** [kubenetworks/kubevpn#47] [Show and tell]
I created a [kubevpn-bin](https://aur.archlinux.org/packages/kubevpn-bin) package for Arch in the Arch AUR.  It's pulling from release zip files and does nothing fancy.  It's the first/only Arch package I've published, if you maintainers would ever like to take that over either to automate updates or just to have it be more official I'm happy to transfer it.  At some point I'll probably create another package to build from git directly instead of use the release binaries (AUR tends to have both "build from source" and "install binaries" packages).

Also, great work on kubevpn, I've wanted a tool like this for so long and it has surprised me that none existed.  It works exactly like I want and is a huge help.  Keep up the great work!

**Comment by wencaiwulue** [comment]
@jonstelly can you update version of Arch AUR. I released new version kubevpn v2.2.9～. Thanks in advance
